db/migrate/20230418074141_add_spam_to_comments.rb

```diff
@@ -0,0 +1,49 @@
+class AddSpamToComments < ActiveRecord::Migration[6.0]
+  def up
+    if Rails.env.staging? || Rails.env.production?
+      database = Comment.connection.current_database
+
+      puts <<~PTOSC
+        Schema Change Command:
+
+        pt-online-schema-change D=#{database},t=comments \\
+          --alter "ADD COLUMN spam BOOLEAN NOT NULL DEFAULT 0" \\
+          --no-drop-old-table \\
+          -uroot --ask-pass --chunk-size=5k --max-flow-ctl 0 --pause-file /tmp/pauseme \\
+          --max-load Threads_running=15 --critical-load Threads_running=100 \\
+          --set-vars innodb_lock_wait_timeout=2 --alter-foreign-keys-method=auto \\
+          --execute
+
+        Table Deletion Command:
+
+        DROP TABLE IF EXISTS `#{database}`.`_comments_old`;
+      PTOSC
+    else
+      add_column :comments, :spam, :boolean, default: 0, null: false
```

Comment:
Looks good! One minor comment, can you use `default: false` here to be consistent with the schema.rb dump format? Like:

https://github.com/otwcode/otwarchive/blob/56ef6bb7390df891814cc1843db627cc80419c39/db/schema.rb#L365

(0 is fine for the `ADD COLUMN` SQL command, however.)

Reply:
Thanks!